src/components/pdScrollArea/index.tsx

```diff
@@ -120,7 +120,7 @@ export const PDScrollArea = polymorphicComponent<'div', IPDScrollAreaProps>(
             className={cn(
               'limechain-scroll-vertical',
               'group/scroll',
-              'peer z-[100]',
```

Comment:
Could this affect the other places where the scroll area is being used? (Just wondering)